applicatn.py

```diff
@@ -34,7 +34,7 @@ def register():
         name = request.form.get("name")
         password = request.form.get("password")
         userdata = Users(username=name, password=password, timestamp=str(datetime.now()))
```

Comment:
let the time get assigned in the class itself, instead of burdening the server

Reply:
Hello Mam,
 They mentioned to write the table separately in assignment task on NGlms.
Hence I rewrote the code to create table.

Reply:
done mam. please check. tq